Kbuild

```diff
@@ -42,4 +42,4 @@ obj-$(CONFIG_LUNATIK_DATA) += lib/luadata.o
 obj-$(CONFIG_LUNATIK_PROBE) += lib/luaprobe.o
 obj-$(CONFIG_LUNATIK_SYSCALL) += lib/luasyscall.o
 obj-$(CONFIG_LUNATIK_XDP) += lib/luaxdp.o
-
```

Comment:
avoid removing lines that aren't related to your patch

Reply:
Got it